tests/Vue/Pages/Results.spec.js

```diff
@@ -3,11 +3,16 @@ import Results from '@/Pages/Results.vue';
 import MismatchesTable from '@/Components/MismatchesTable.vue';
 
 import { ReviewDecision } from '@/types/Mismatch.ts';
+import axios from 'axios';
 
 // Stub the inertia vue components module entirely so that we don't run into
 // issues with the Head component.
 jest.mock('@inertiajs/inertia-vue', () => ({}));
 
+jest.mock("axios", () => ({
+    put: jest.fn()
+  }));
```

Comment:
```suggestion
}));
```
The nit has been picked XD